.changelog/2777.feature.md

```diff
@@ -0,0 +1,6 @@
+go/staking: include LastBlockFees in genesis
```

Comment:
oh upgrade this to breaking

Reply:
hm so not sure if this is the case, under the "breaking" label in github we have a note: "Breaking change requiring state wipe", i don't think that is the case here, also a genesis without last_block_fees does still work. However the change does break forward compatibility as genesis files with last_block_fees field would not work with old versions :thinking:

Reply:
oh ok